crates/typst-cli/src/compile.rs

```diff
@@ -308,16 +309,11 @@ fn open_file(open: Option<&str>, path: &Path) -> StrResult<()> {
 
 /// Print diagnostic messages to the terminal.
 pub fn print_diagnostics(
-    world: &SystemWorld,
+    world: &mut SystemWorld,
```

Comment:
why this change?

Reply:
This appears to have been a mistake that occured while rebasing.